src/test/java/org/jitsi/meet/test/web/WebTestBase.java

```diff
@@ -238,7 +270,12 @@ private Participant joinParticipant(
 
             String configPrefix = "web.participant" + (index + 1);
 
-            p = participants.createParticipant(configPrefix);
+            if (options == null)
```

Comment:
The logic for providing default ParticipantOptions should be in shared code. One way fo doing it would be to move it to the ParticipantHelper.createParticipant and allow null argument.